jentool/__init__.py

```diff
@@ -94,6 +94,12 @@ def _parser():
     parser_nodes_list = subparsers.add_parser(
         'nodes-list', help='List all nodes')
     parser_nodes_list.set_defaults(func=nodes_list)
+    # jobs failing
+    parser_jobs_failing = subparsers.add_parser(
+        'jobs-failing', help='List failing jobs')
+    parser_jobs_failing.add_argument('pattern', metavar='pattern', help='the Jenkins job name(s) (regex)')
+    parser_jobs_failing.add_argument('--max-score', '-m', default=0, help='the maximum health score to look for')
```

Comment:
sorry for another round - but you can use `type=int` so you don't need the cast later